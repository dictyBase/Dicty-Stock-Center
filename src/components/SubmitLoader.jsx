import React, { Component } from 'react'
import 'styles/core.scss'

export default class SubmitLoader extends Component {
    displayName = 'loading component during submitting the order form';
    render() {
        return (
            <div className="container">
              <h2> Feel the bern while your information is being submitted........ </h2>
            </div>
        )
    }
}
