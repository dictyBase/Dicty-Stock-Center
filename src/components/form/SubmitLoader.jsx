import React, { Component } from 'react'
import { Container } from 'styles'
import { Flex, Box } from 'rebass'
import FontAwesome from 'react-fontawesome'
import { CenteredBox } from 'styles'

export default class SubmitLoader extends Component {
    displayName = 'loading component'
    render() {
        return (
            <Container>
                <Flex wrap justify="center">
                    <CenteredBox>
                        <h1>Please wait...</h1>
                        <FontAwesome name="spinner" size="5x" pulse mw />
                    </CenteredBox>
                </Flex>
            </Container>
        )
    }
}
