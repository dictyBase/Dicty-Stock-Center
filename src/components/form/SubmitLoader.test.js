import React from 'react'
import renderer from 'react-test-renderer'
import SubmitLoader from './SubmitLoader'

test('matching a snapshot of SubmitLoader', () => {
    const component = renderer.create(
    <SubmitLoader />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
