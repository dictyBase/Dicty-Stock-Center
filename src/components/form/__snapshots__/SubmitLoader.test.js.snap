// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot of SubmitLoader 1`] = `
.c0 {
  padding-right: 15px;
  padding-left: 15px;
  margin-right: auto;
  margin-left: auto;
}

.c1 {
  text-align: center;
}

@media (min-width:768px) {
  .c0 {
    width: 750px;
  }
}

@media (min-width:992px) {
  .c0 {
    width: 970px;
  }
}

@media (min-width:1200px) {
  .c0 {
    width: 1170px;
  }
}

<div
  className="c0"
>
  <div
    className="sc-htpNat eDtRyC sc-bdVaJa iHZvIS"
    is={null}
  >
    <div
      className="c1 sc-bdVaJa iHZvIS"
      is={null}
    >
      <h1>
        Please wait...
      </h1>
      <span
        aria-hidden={true}
        className="fa fa-spinner fa-5x fa-pulse"
        mw={true}
      />
    </div>
  </div>
</div>
`;
